patchwork-extensions-bakedmodel/src/main/java/net/patchworkmc/impl/extensions/bakedmodel/ForgeBlockModelRenderer.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.patchworkmc.impl.extensions.bakedmodel;
+
+import java.util.Random;
+
+import net.minecraftforge.client.model.data.IModelData;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.client.render.BufferBuilder;
+import net.minecraft.client.render.block.BlockModelRenderer;
+import net.minecraft.client.render.model.BakedModel;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.BlockRenderView;
+
+public interface ForgeBlockModelRenderer {
+	void patchwork$tesselate_ModelData(IModelData modelData);
+	void patchworl$tesselateSmoothFlat_ModelData(IModelData modelData);
```

Comment:
Is this getting? setting? an action? Could use some documentation.

Reply:
These function are used to setup the context, which optionally contains an IModelData param. I will add some comments.